frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -1288,6 +1288,7 @@ VAR__DETAIL__NOT_MEMORY_PINNED_NUMA=$detailMessage cannot accommodate memory of
 VAR__DETAIL__NO_SUITABLE_VF=$detailMessage there are no free virtual functions which are suitable for virtual nic(s) ${vnicNames}. A virtual function is considered as suitable if the VF's configuration of its physical function contains the virtual nic's network/network label
 VAR__DETAIL__NOT_PINNED_TO_HOST=$detailMessage the host is not present in the pinning list.
 VAR__DETAIL__MISSING_VDS_HOOK=$detailMessage the host does not contain required VDS hook ${vdsHook}
+VAR__DETAIL__NO_HOST_DATA=$detailMessage the host has no CPU topology data.
```

Comment:
how about renaming this to VAR__DETAIL__NO_HOST_CPU_DATA and the phrasing to something like: "the CPU topology of the host has not been retrieved"